test/cluster_test.lua

```diff
@@ -0,0 +1,241 @@
+local t = require('luatest')
+local cbuilder = require('luatest.cbuilder')
+local cluster = require('luatest.cluster')
+local utils = require('luatest.utils')
+local fio = require('fio')
+
+local g = t.group()
+
+local root = fio.dirname(fio.abspath('test.helpers'))
+
+-- These are extra server opts passed to the cluster.
+-- They are needed for the server to be able to access
+-- luatest.coverage.
+local server_opts = {
+    env = {
+        LUA_PATH = root .. '/?.lua;' ..
+            root .. '/?/init.lua;' ..
+            root .. '/.rocks/share/tarantool/?.lua',
+    }
+}
+
+local function assert_instance_running(c, instance, replicaset)
```

Comment:
Do you consider [adjusting](https://github.com/tarantool/tarantool/blob/5ae2263852dc0630c6b531492bc6598b849f9d7c/.luacheckrc#L14-L23) `.luacheckrc`?

Reply:
At least we can do it for a particular file.

Reply:
I suggest not to touch `.luacheckrc` without specific reason. Current test code seems quite easy to read.

Reply:
OK.